docs/docs/code/nodejs/http-requests/README.md

```diff
@@ -490,87 +490,11 @@ export default defineComponent({
 
 [Copy this workflow](https://pipedream.com/@dylburger/stream-a-file-upload-p_6lC1d2Z/edit) to run this example.
 
-## Use an HTTP proxy to proxy requests through another host
-
-When you make HTTP requests to certain services, they might require you whitelist a set of IP addresses those requests come from. Often, this is to improve the security of the target service.
```

Comment:
This section I agree we can remove, and instead tell people here that they can use VPCs to send requests from a static IP, but this example code helps if they're running their own HTTP proxy or connecting to a third-party proxt (something I've seen some customers do).